### PR TITLE
[DependencyInjection] fix tests

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 6.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Issues        | see the test failures after #53876
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/Tests/Fixtures/ReadOnlyClass.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 namespace Symfony\Component\DependencyInjection\Tests\Fixtures;
 
-readonly class ReadonlyTest
+readonly class ReadOnlyClass
```
Comment (xabbuh) [inline src/Symfony/Component/DependencyInjection/Tests/Fixtures/ReadOnlyClass.php:14]: class renamed so that it is not being eagerly loaded by PHPUnit

**Comment by nicolas-grekas** [comment]
But readonly classes are supported since 8.2, no?

**Comment by xabbuh** [comment]
They are, but the VarExporter component doesn't on 8.2 (probably because of https://wiki.php.net/rfc/readonly_amendments, see #49282).

**Comment by nicolas-grekas** [comment]
Thank you @xabbuh.

**Pull request merged**